pkg/windows/windows.go

```diff
@@ -136,15 +139,18 @@ var (
 		cniDir,
 		CniConfDir,
 		logDir,
+		KubeletLogDir,
 		KubeProxyLogDir,
 		wicdLogDir,
 		HybridOverlayLogDir,
 		ContainerdDir,
-		ContainerdLogDir}
+		ContainerdLogDir,
+		podManifestDirectory,
+	}
 )
 
 // getFilesToTransfer returns the properly populated filesToTransfer map
-func getFilesToTransfer() (map[*payload.FileInfo]string, error) {
+func (vm *windows) getFilesToTransfer() (map[*payload.FileInfo]string, error) {
```

Comment:
Why is this part of the `windows` struct? Seems to be an independent helper.

Reply:
Bootstrap files field was added to the `windows` struct and are accessed within this method

Reply:
I am not sure why that is needed.